lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -377,6 +379,27 @@ class DetailsRenderer {
     return element;
   }
 
+  /**
+   * @param {LH.Audit.Details.SourceLocationValue} item
+   * @return {Element}
+   * @protected
+   */
+  renderSourceLocation(item) {
+    let element;
+    if (item.canLink) {
```

Comment:
so `canLink` means it can be linked as if it were a real URL?

Reply:
yup